src/otx/algo/detection/heads/delta_xywh_bbox_coder.py

```diff
@@ -8,6 +8,8 @@
 import torch
 from torch import Tensor
 
+from otx.algo.detection.utils.utils import clip_bboxes_export
```

Comment:
Is there any special reason to rename this?

Reply:
In the data pipeline, there is `clip_bboxes` as well.
I renamed this function to distinguish functions for the next step, refactoring duplicated modules.